components/contributors/GithubActivity.tsx

```diff
@@ -41,11 +47,16 @@ let renderText = (activity: Activity) => {
           <div className="mt-2 text-foreground">
             <p className="break-words">{activity["text"]}</p>
           </div>
-        </div>
+        </motion.div>
       );
     case "comment_created":
       return (
-        <div className="min-w-0 flex-1">
+        <motion.div
+          initial={{ opacity: 0, x: -30 }}
+          whileInView={{ opacity: 1, x: 0 }}
+          transition={{ delay: 0.3, type: "tween" }}
+          className="min-w-0 flex-1"
+        >
```

Comment:
1. Let's keep the animation consistent for all types of events.
2. Move this to the parent component instead of redefining the animation for each event type.

Reply:
working on it 👍